e2e/webpack_devserver/serve_test.sh

```diff
@@ -16,13 +16,16 @@ _sedi() {
 
 echo "TEST - $0: $1"
 
-./node_modules/.bin/ibazel run "$1" "$BZLMOD_FLAG" >/dev/null 2>&1 &
+ibazel_logs=$(mktemp)
+echo "Capturing ibazel logs to $ibazel_logs"
+./node_modules/.bin/ibazel run "$1" "$BZLMOD_FLAG" >"$ibazel_logs" 2>&1 &
 ibazel_pid="$!"
 
 function _exit {
-    echo "Cleanup..."
```

Comment:
Why drop the logging?

Reply:
When there is an "ERROR" it is confusing that the error output is not the last line in the test log